ftml/src/data/user_info.rs

```diff
@@ -53,10 +56,9 @@ impl UserInfo<'_> {
         UserInfo {
             user_id: 0,
             user_name: cow!("michal-frackowiak"),
+            user_karma: 5,
             user_avatar_data: cow!(AVATAR_BASE64_DATA),
-            user_profile_url: cow!(
-                "https://www.wikijump.com/user:info/michal-frackowiak",
-            ),
+            user_profile_url: cow!("/user:info/michal-frackowiak",),
```

Comment:
Rustfmt is conservative and doesn't change punctuation in macros since it doesn't know if it matters or not.
```suggestion
            user_profile_url: cow!("/user:info/michal-frackowiak"),
```